frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -93,4 +96,37 @@ const ProjectOverview: FC<{
     );
 };
 
+const NewProjectOverview: FC<{
+    storageKey?: string;
+}> = ({ storageKey = 'project-overview-v2' }) => {
+    const projectId = useRequiredPathParam('projectId');
+    const projectName = useProjectOverviewNameOrId(projectId);
+
+    // TODO: remove this dependency. We only need environment names
+    const { project } = useProjectOverview(projectId, {
```

Comment:
from this API we only care about project.environments.name so we can e.g. expose it in project insights